app/public/js/profile.js

```diff
@@ -106,12 +106,16 @@ function getCountryElement()
 
 function updateRegionOptions()
 {
-	var country_id = parseInt(getCountryElement().val());
-	var $datalist = $('#regions');
-	$datalist.empty();
+	let country_id = parseInt(getCountryElement().val());
+	let home_region = $('#home_region');
```

Comment:
Keep the $ for jQuery objects like that home_region.  Rename home_region to $home_region.

The naming convention is explained in more detail at:
https://stackoverflow.com/questions/10204606/is-there-any-specific-reason-behind-using-with-variable-in-jquery